R/share.R

```diff
@@ -42,22 +42,28 @@ setMethod("is.editor", "PermissionTuple", function (x) {
 #' of the number of emails given.
 #' @param notify logical: should users who are getting new privileges on this
 #' dataset be sent an email informing them of this fact? Default is
+#' @param message character: what message should users who are getting notified
+#' about new permissions on this dataset receive?
```

Comment:
Move this new code down a line. L47 goes with L44. Also indicate the default (no message).